projects/cds/src/cds-config-validator.ts

```diff
@@ -0,0 +1,20 @@
+import { CdsConfig } from './cds-config';
+
+export function cdsConfigValidator(config: CdsConfig) {
```

Comment:
Please add a return type for this method

Reply:
We cannot see a return type on any of the other validators. I guess we could add a return type of string and then in the case there are no validation problems we return a falsey result so that the error is not shown to the user?

Reply:
Yes, I believe returning `null` should do the trick. In that case `string | null` is the appropriate return type (or maybe `undefined` instead of `null`, whichever works the best).

Reply:
Changed to return string | void, as per the ConfigValidator in core